app/uk/gov/hmrc/customsservicestatus/models/State.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customsservicestatus.models
+
+import play.api.libs.json.Json
+case class State(state: String)
+object State {
```

Comment:
Minor: can we put at least a line of whitespace between the `import` and the definitions below? 

Ideally, it would also be good to put at least a line of whitespace between, say, a class and an object definition.

Reply:
done